src/index.html

```diff
@@ -1,14 +1,105 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect"
+      href="https://fonts.googleapis.com"
+     >
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="crossorigin"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Roboto:wght@100;400;500&display=swap"
+      rel="stylesheet"
+    >
+    <title>Mayo header</title>
+    <link rel="stylesheet"
+      href="/src/style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+          href="#">
+        <img src="/src/images/logo.png"
+         alt="Logo"
+          class="logo"
+        >
+      </a>
+      <nav class="nav-menu">
+        <ul class="menu">
+          <li class="menu-item">
+            <a
+              href="#"
+              title="Click to visit link"
+              class="menu-link is-active "
+            >
+            Apple
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              title="Click to visit link"
+              class="menu-link"
+            >
+            Samsung
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              title="Click to visit link"
+              class="menu-link"
+            >
+            Smartphones
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              title="Click to visit link"
+              data-qa="hover"
+              class="menu-link"
+              >
+              Laptops &#38 Computers
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              title="Click to visit link"
+              class="menu-link"
+            >
+            Gadgets
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#" title="Click to visit link"
+              class="menu-link"
+            >
+            Tablets
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+            href="#" title="Click to visit link"
+            class="menu-link"
+            >
+            Photo
```

Comment:
code it not aligned
check other places
```suggestion
            <a
              href="#" title="Click to visit link"
              class="menu-link"
            >
              Photo
```